### ISSUE TITLE
sample-apiserver does not have script or instructions for building

### ISSUE THREAD

**Issue by MikeSpreitzer** [kubernetes/kubernetes#73390]
**What happened**:
I wanted to test the latest version of the sample-apiserver.  I still did not find instructions or a script for building it.  I found the `build-image.sh` script, but that relies on the `_output` directory produced when building k/k.

**What you expected to happen**:
I expected this sample to have standalone build instructions or scripting.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by MikeSpreitzer** [comment]
@kubernetes/sig-api-machinery-bugs

**Comment by MikeSpreitzer** [comment]
I created a script of my own to build it:
```
#!/bin/bash

GOOS=linux GOARCH=amd64 go build
cp sample-apiserver artifacts/simple-image/kube-sample-apiserver
IMGREF=${DOCKER_PREFIX:-$LOGNAME}/kube-sample-apiserver:$(git rev-parse HEAD)
docker build -t $IMGREF ./artifacts/simple-image
docker push $IMGREF
```

**Comment by MikeSpreitzer** [comment]
Also, the existing `hack/update-codegen.sh` is not designed to be invoked by a developer who has done `go get k8s.io/sample-apiserver` but rather is designed to work only in the context of a full build of k/k.

**Comment by MikeSpreitzer** [comment]
/remove-lifecycle stale
Being addressed by #73447
